setup.py

```diff
@@ -5,42 +5,44 @@
 # -----------------------------------------------------------------------------
 
 import os
-import codecs
+from codecs import open
 from setuptools import setup, find_packages
 
 
 here = os.path.abspath(os.path.dirname(__file__))
-with codecs.open(os.path.join(here, 'description.txt'), encoding='utf-8') as f:
+with open(os.path.join(here, 'description.txt'), encoding='utf-8') as f:
```

Comment:
codecs.open didn't cover up the reserved open

Reply:
fixed.